packages/services/f-analytics/README.md

```diff
@@ -21,13 +21,11 @@ This component abstracts away the gathering of the various data values needed fo
 
 ## Benefits (Now)
 - Single point to record GA data: Currently we have GA/GTM logic scattered throught various features, this will allow that logic to be removed and centralise it a single component.
-- Self sufficient: With only supplying a small amount of global data this component will attempt to evaluate, gather and record all the data required for the GA `platformData` model.  Once registered it will perform the pushing of the GA `platformData` model on the correct event/hook.
-- Reactive: When the route changes (i.e. the page changes) it will re-evaluate and re-pushes the GA `platformData` model.
+- Self sufficient: With only supplying a small amount of global data this component will attempt to evaluate, gather and record all the data required for the GA `platformData` and `userData` model.
```

Comment:
Small typo:
```suggestion
- Self sufficient: With only supplying a small amount of global data this component will attempt to evaluate, gather and record all the data required for the GA `platformData` and `userData` models.
```

Reply:
@AndyMayJE  as you will add info about your changes in your PR, could you please fix this small typo as well?

Reply:
@dandel10n of course I have left the readme until this or is merged